cachito/web/static/api_v1.yaml

```diff
@@ -277,6 +277,75 @@ paths:
                 $ref: "#/components/schemas/RequestConfiguration"
       security:
       - "Kerberos Authentication": []
+  "/requests/{id}/content-manifest":
+    get:
+      description: Return the content manifest of a request
+      parameters:
+      - name: id
+        in: path
+        required: true
+        description: The ID of the Cachito request to retrieve the content manifest for
+        schema:
+          type: integer
+      responses:
+        "200":
+          description: The content manifest of the Cachito request
+          content:
+            application/json:
+              schema:
+                type: object
+                nullable: true
+                properties:
+                  metadata:
+                    type: object
+                    properties:
+                      icm_version:
+                        type: integer
+                        example: 1
+                      icm_spec:
+                        type: string
+                        example: "https://json-schema.icm.example.com"
+                      image_layer_index:
+                        type: integer
+                        example: 1
+                  image_contents:
+                    type: array
+                    items:
+                      type: object
+                      properties:
+                        purl:
+                          $ref: "#/components/schemas/Purl"
+                        dependencies:
+                          type: array
+                          items:
+                            type: object
+                            properties:
+                              purl:
+                                $ref: "#/components/schemas/Purl"
+                        sources:
+                          type: array
+                          items:
+                            type: object
+                            properties:
+                              purl:
+                                $ref: "#/components/schemas/Purl"
+                  content_sets:
```

Comment:
Will Cachito ever set this? If not, I think this should be removed.

Reply:
Not really, I will remove this one

Reply:
removed